modules/@apostrophecms/page-type/index.js

```diff
@@ -170,6 +170,24 @@ module.exports = {
           }
         }
       },
+      afterRevertPublishedToPrevious: {
+        async replayMoveAfterRevert(req, result) {
+          const publishedReq = {
+            ...req,
+            mode: 'published'
+          };
+          if (!result.published.level) {
```

Comment:
Can we use `=== 0`? It communicates the goal better. If it was undefined that'd be another interesting situation, no (though probably unlikely)?